shared/stats/src/index.mjs

```diff
@@ -44,12 +56,31 @@ export async function getStats (modulePath) {
   // Modules bundled
   const modules = stats?.[base] ?? {}
 
+  const moduleCount = Object.keys(modules).length
+
   // Modules total size
   const total = Object.values(modules)
     .map(({ rendered }) => rendered)
     .reduce((total, rendered) => total + rendered, 0)
 
-  return { total, modules }
+  return { total, modules, moduleCount }
+}
+
+/**
+ * Returns file sizes of key files
+ *
+ * @returns {Promise<{[key: string]: import('fs').Stats}>} - File names and size
+ */
+export async function getFileSizes () {
+  /** @type { { [key: string]: import('fs').Stats } } */
```

Comment:
Anyway to rewrite this without this messy type annotation?

Reply:
Yeah, using the same array mapping idea to preserve types?

```mjs
const fileTasks = filesForAnalysis
  .map(async (filename) => /** @type {const} */(
    [filename, await stat(join(paths.root, filename))]
  ))

return Promise.all(fileTasks)
  .then(Object.fromEntries)
```